windows/test-app.js

```diff
@@ -96,6 +95,32 @@ function copyAndReplace(
   }
 }
 
+function copyTestProject(
+  testProjectFilePath,
+  testProjectTemplateFile,
+  srcRootPath,
+  projDir,
+  projectFilesDestPath
+) {
+  const testProjectFileReplacements = {
+    "\\$\\(SourceFilesPath\\)": path.relative(
+      testProjectFilePath,
+      path.join(srcRootPath, projDir)
+    ),
+    "\\$\\(ReactTestAppProjectPath\\)": path.relative(
+      testProjectFilePath,
+      projectFilesDestPath
+    ),
+  };
+
+  copyAndReplace(
+    path.join(testProjectFilePath, testProjectTemplateFile),
+    testProjectFilePath,
+    "ReactTestAppTests.vcxproj",
```

Comment:
Can we parameterize this? Maybe we should split this in two PRs? One for adding the CI, the second to add tests?